README.md

```diff
@@ -76,11 +79,30 @@ function `infotodict`, which takes a single argument `seqinfo`.
 
 ### `seqinfo` and the `s` variable
 
-Each item in `seqinfo` contains the following ordered elements.
+`seqinfo` is a list of namedtuple objects, each containing the following fields:
+
+* total_files_till_now
+* example_dcm_file
+* series_number
+* unspecified1
```

Comment:
if i merge #56 first this `unspecified1` will need to change. i'm just waiting on travis for the moment.